tests/unit/certGen.unitTest.ts

```diff
@@ -2093,6 +2094,462 @@ describe("cert-gen", () => {
         });
     });
 
+    context("CertGenService for ADR", () => {
+        context("when a passing test result for ADR is read from the queue", () => {
+            const event: any = lodash.cloneDeep(queueEventPass);
+            const testResult: any = JSON.parse(event.Records[1].body);
+            testResult.testTypes.testTypeId = "50";
+
+            context("and a payload is generated", () => {
+                context("and no signatures were found in the bucket", () => {
+                    it("should return an ADR_PASS payload without signature", () => {
+                        const expectedResult: any = {
+                            Watermark: "NOT VALID",
+                            ADR_DATA: {
+                                ChasisNumber: "P012301098765",
+                                ApplicantDetails: {
+                                    name: "string",
+                                    street: "string",
+                                    town: "string",
+                                    city: "string",
+                                    postcode: "string"
+                                },
+                                PermittedDangerousGoods: [ "FP <61 (FL)" ],
+                                TankManufacturer: "string",
+                                Tc2InitApprovalNo: "string",
+                                TankManufactureSerialNo: "string",
+                                YearOfManufacture: "string",
+                                TankCode: "string",
+                                SpecialProvisions: "string",
+                                TankStatement: {
+                                    substancesPermitted: "Substances permitted under the tank code and any special provisions specified in 9 may be carried",
+                                    statement: "string",
+                                    productList: "string"
+                                },
+                                RegistrationNumber: "VM14MDT",
+                                VehicleType: "string",
+                                ExpiryDate: "2020-02-25T15:29:40.032Z",
+                                AtfNameAtfPNumber: "Abshire-Kub 09-4129632" },
+                            Signature: {
+                            ImageType: "png",
+                                ImageData: null
+                            }
+                        };
+
+                        const techRecordResponseMock = {
```

Comment:
Same as above. Adding in resources will reduce the lines of code in the unit test.

Reply:
fixed

Reply:
agreed. These are huge - chuck them into a json file for  re-use